README.md

```diff
@@ -345,14 +345,14 @@ _OR_
 <pre>
 rsyslog::log_server:
   - 'log1'
-  - 'log2'
+  - 'log2:1514'
```

Comment:
Heh, that was a typo anyway, it should have been log3 - implying a third log server destination using a non-default port.